spec/system/admin/product_import_spec.rb

```diff
@@ -53,6 +53,14 @@
 
   let(:shipping_category_id_str) { Spree::ShippingCategory.all.first.id.to_s }
 
+  let(:available_units) {
+    ["mg", "g", "kg", "T", "oz", "lb", "mL", "cL", "dL", "L", "kL", "gal"].join(",")
+  }
+
+  before do
+    Spree::Config.set_preference(:available_units, available_units)
+  end
```

Comment:
We should actually mock the preference, so that it doesn't leak into other specs. In fact, there's and example below in this file!
It wasn't working because you used a different method for accessing the preference. I've added a commit to change this to use the more standard format (`Spree::Config.available_units`), and now `product_import_spec.rb` works without any further changes.